tests/conftest.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import os
+from datetime import datetime
+from pathlib import Path
+
+import pytest
+from tests.test_config import RESULTS_DIR, TEST_ID, SCYLLA_DOCKER_IMAGE
+
+from ccmlib.scylla_docker_cluster import ScyllaDockerCluster
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session")
+def results_dir():
+    LOGGER.info("Creating test directory...")
+    dir_name = "tests" / Path(RESULTS_DIR)
+    dir_name.mkdir(exist_ok=True)
+    return dir_name
+
+
+@pytest.fixture(scope="session")
+def test_id():
+    ccm_test_id = TEST_ID or datetime.now().strftime("%Y%m%d-%H%M%S")
+    LOGGER.info("Using test id: '{}'".format(ccm_test_id))
+    return ccm_test_id
+
+
+@pytest.fixture(scope="session")
+def test_dir(test_id, results_dir):
+    max_test_dirs, dir_count = 1, 100
```

Comment:
why all this `max_test_dirs` ? why can't we just use one directory ?

Reply:
@fruch 
I copied the code from Siren, it is meant to limit the number of directories - we do not want to have a lot of directories on the computer.
If will be free time, the idea is that it will be possible to stop the test and use the same cluster in the next run as well (assuming the cluster is up).